### DISCUSSION TITLE
[Feature Request] New site -> Open in IDE

### DISCUSSION THREAD

**Discussion by avenjamin** [beyondcode/herd-community#776] [Ideas]
When creating a new site in Herd there are two buttons on the final screen:

1. Open in browser
2. Close

I'd like to propose an additional option:

3. Open in IDE

Further to this a forth option might be:

4. Show in Finder / Explorer

**Comment by mpociot** [comment]
Yes! We are currently working on a new UI for the sites list which will offer a lot more functionality like this 🙂

**Reply by avenjamin** [reply]
Awesome! I look forward to it - keep up the great work 😃
